test-framework/src/main/java/org/apache/brooklyn/test/framework/TestHttpCallImpl.java

```diff
@@ -109,6 +130,49 @@ public Integer get() {
         }
     }
 
+    private HttpRequestBase createHttpMethod(HttpMethod method, String url, Map<String, String> headers, String body) throws Exception {
+        switch (method) {
+            case GET:
+                HttpTool.HttpGetBuilder httpGetBuilder = new HttpTool.HttpGetBuilder(new URI(url));
```

Comment:
Agree but the the `HttpTool.HttpRequestBuilder` is based on the request class which is why there is this conversion enum -> request. Do you think the `HttpMethod` and this private method `createHttpMethod` should move into `HttpTool`?